lfs/ssh.go

```diff
@@ -37,12 +38,22 @@ func sshAuthenticate(endpoint Endpoint, operation, oid string) (sshAuthResponse,
 
 	cmd := exec.Command(exe, args...)
 
-	out, err := cmd.CombinedOutput()
+	// Save stdout and stderr in separate buffers
+	var outbuf, errbuf bytes.Buffer
+	cmd.Stdout = &outbuf
+	cmd.Stderr = &errbuf
```

Comment:
Never mind, I see it's OK before Start(). I've just never seen that approach before, useful :)